mdata/cache/accnt/if.go

```diff
@@ -0,0 +1,24 @@
+package cache
+
+type Accnt interface {
+	Add(string, uint32, uint64)
+	Hit(string, uint32)
+	GetEvictQ() chan *EvictTarget
+}
+
+type EvictTarget struct {
+	Metric string
+	Ts     uint32
+}
+
+func NewAccnt(maxSize uint64) Accnt {
```

Comment:
i see there's a few places where we allocate a FlatAccnt struct and then call init() on it. by turning this into the NewFlatAccnt constructor, the logic from init can also go in there